console/console-format-specifier-symbol-throws.any.js

```diff
@@ -0,0 +1,53 @@
+"use strict";
+// https://console.spec.whatwg.org/#formatter
+
+const clauses = ["must throw a TypeError when applying the", "format specifier on a Symbol"];
+
+function errorTag(subs, clause1, clause2) {
+  return subs[0] + clause1 + subs[1] + clause2;
+}
+
+test(() => {
+  assert_throws({name: "TypeError"}, () => {
+    console.log("%i", Symbol.for("description"));
+  }, errorTag`log() ${clauses[0]} %i ${clauses[1]}`);
```

Comment:
Can't you just use untagged template strings for these?

Reply:
Haha was just going to do this. Sitting in class thinking "Hm I made that more complicated than it needs to be"

Reply:
If the goal is really to DRY this up then you could reduce it all down to

```js
assertFailureCase("%i", "log")
```